### PR TITLE
added product description feature file for AZ-57 story

### PR DESCRIPTION
**Pull request by abhilashawagh**
Hello Ashwini,

    Please review the newly added feature file for AZ-57 story and let me know the changes if any.

Thanks,
Abhilasha Wagh.

### CODE REVIEW

File: src/test/resources/features/prodDesc.feature [hunk 1]
```diff
@@ -0,0 +1,23 @@
+@ProductDesc
+Feature: Amazon health check
+  
+  @tag1
+  Scenario: Product description for amazon---$#
```
Comment (AshwiniPKadam) [inline src/test/resources/features/prodDesc.feature]: remove '---$#' from scenario

File: src/test/resources/features/prodDesc.feature [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+    Given I want to write a step with precondition
+    And some other precondition
+    When I complete action
+    And some other action
+    And yet another action
+    Then I validate the outcomes
+    And check more outcomes
+
+  @tag-#RT
```
Comment (AshwiniPKadam) [inline src/test/resources/features/prodDesc.feature]: remove '-#RT' from tag

**Review by AshwiniPKadam — COMMENTED** [review]
please do corresponding changes in feature file.

File: src/test/resources/features/prodDesc.feature (hunk 1 above)
Comment (abhilashawagh) [reply src/test/resources/features/prodDesc.feature]: made changes in Scenario line

File: src/test/resources/features/prodDesc.feature (hunk 2 above)
Comment (abhilashawagh) [reply src/test/resources/features/prodDesc.feature]: Made changes in tag line

**Comment by AshwiniPKadam** [comment]
Alright, now you can merge feature file to master branch.

**Pull request merged**